### PR TITLE
Custom reports added

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Add few custom report:

- All Authentications
- Email registration
- SMS registration
- Sponsor registration
- IPv4 Archive
- IPv4 History

# Impacts
Custom report itself 

# Delete branch after merge
YES 

# NEWS file entries
## New Features
* Default reports. Add view on all authentication types (SMS, Email, Sponsor,etc..)

### CODE REVIEW

**Comment by lzammit** [comment]
Will be re-worked to put all those reports into a conf/report.conf.default

**Comment by julsemaan** [comment]
Finished putting the reports in the admin.

@dwlfrth, can you take a quick look and merge please

File: conf/report.conf.defaults [hunk 1]
```diff
@@ -0,0 +1,80 @@
+[authentications]
```
Comment (jrouzierinverse) [inline conf/report.conf.defaults:1]: Does it make sense to add a copyright notice in our config?

Comment (julsemaan) [reply conf/report.conf.defaults:1]: not really IMO

**Comment by dwlfrth** [comment]
OK to be merged ?

**Pull request merged**